nixos/modules/tasks/network-interfaces-scripted.nix

```diff
@@ -263,6 +268,10 @@ let
               ip link del "${n}" || true
               rm -f /run/${n}.interfaces
             '';
+            preStop = ''
+              # Save currently attached slaves
+              ip link | grep master | awk '{print $2,$9}' | grep "${n}" | cut -d ":" -f 1 > /run/${n}.slaves
```

Comment:
```grep -F``` for non-regex match.
interface names might have characters which could be understood as regexp specials